edgelet/build/windows/iotedge.wm.xml

```diff
@@ -14,8 +14,8 @@
     <file source="$(_REPO_ROOT)\edgelet\target\$(_Arch)\release\iotedged.pdb" destinationDir="$(runtime.programFiles)\iotedge" />
     <file source="$(_REPO_ROOT)\edgelet\target\$(_Arch)\release\iotedged_eventlog_messages.dll" destinationDir="$(runtime.programFiles)\iotedge" />
     <file source="$(_REPO_ROOT)\edgelet\hsm-sys\azure-iot-hsm-c\build\Release\iothsm.dll" destinationDir="$(runtime.programFiles)\iotedge" />
-    <file source="$(_OPENSSL_ROOT_DIR)\bin\libeay32.dll" destinationDir="$(runtime.programFiles)\iotedge" />
-    <file source="$(_OPENSSL_ROOT_DIR)\bin\ssleay32.dll" destinationDir="$(runtime.programFiles)\iotedge" />
+    <file source="$(_OPENSSL_ROOT_DIR)\bin\libcrypto-1_1-$(_OPENSSL_DLL_SUFFIX).dll" destinationDir="$(runtime.programFiles)\iotedge" />
```

Comment:
why does the new build require the suffix when the old one didn't?

Reply:
openssl sets the filenames, so we have to use what it gives us. If you remember even in your local amd64 build that there was an -x64 suffix.